github-electron/github-electron.d.ts

```diff
@@ -1101,7 +1115,12 @@ declare namespace Electron {
 		 * @param progress Valid range is [0, 1.0]. If < 0, the progress bar is removed.
 		 * If greater than 0, it becomes indeterminate.
 		 */
-		setProgressBar(progress: number): void;
+		setProgressBar(progress: number, options?: {
+			/**
+			 * Mode for the progres bar (Windows).
```

Comment:
I think noting with 'Note: This API is available only on Windows.' is more consistent.